Makefile.pre.in

```diff
@@ -844,7 +845,14 @@ $(WASM_STDLIB): $(srcdir)/Lib/*.py $(srcdir)/Lib/*/*.py \
 python.html: Programs/python.o $(LIBRARY_DEPS) $(WASM_STDLIB)
 	$(LINKCC) $(PY_CORE_LDFLAGS) $(LINKFORSHARED) -o $@ Programs/python.o \
 	    $(BLDLIBRARY) $(LIBS) $(MODLIBS) $(SYSLIBS) \
-	    -s ASSERTIONS=1 --preload-file $(WASM_ASSETS_DIR)
+	    -s ASSERTIONS=1 -s ALLOW_MEMORY_GROWTH=1 \
+		--preload-file $(WASM_ASSETS_DIR)
+
+python.js: Programs/python.o $(LIBRARY_DEPS) $(WASM_STDLIB)
+	$(LINKCC) $(PY_CORE_LDFLAGS) $(LINKFORSHARED) -o $@ Programs/python.o \
+	    $(BLDLIBRARY) $(LIBS) $(MODLIBS) $(SYSLIBS) \
+	    -s ASSERTIONS=1 -s NODERAWFS=1 -s ALLOW_MEMORY_GROWTH=1 \
+		-s EXIT_RUNTIME=1
```

Comment:
- I think that you can drop the dependency on ``$(WASM_STDLIB)``. The build target does not use ``$(WASM_ASSETS_DIR)`` bundle.
- The ``python.html`` target also generated a ``python.js`` file. Would it make sense to use a different name for the target, e.g. ``python-rawfs.js`` ?

```suggestion
python.js: Programs/python.o $(LIBRARY_DEPS)
	$(LINKCC) $(PY_CORE_LDFLAGS) $(LINKFORSHARED) -o $@ Programs/python.o \
	    $(BLDLIBRARY) $(LIBS) $(MODLIBS) $(SYSLIBS) \
	    -s ASSERTIONS=1 -s ALLOW_MEMORY_GROWTH=1 \
	    -s NODERAWFS=1 -s EXIT_RUNTIME=1
```